File: 217-ContainsDuplicate/217-ContainsDuplicate.py
# Last updated: 01/09/2025, 01:05:05
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        if len(set(nums))==len(nums):
            return False
        for i in nums:
            if nums.count(i)>=2:
                return True
        return False

File: 217-ContainsDuplicate/217-ContainsDuplicate.cpp
// Last updated: 01/09/2025, 01:04:59
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        unordered_map<int,int>unmap;
        for(auto i : nums){
            unmap[i]++;
        }
        for(auto i : unmap){
            if(i.second > 1) return true;
        }
        return false;
    }
};